storage/integration_test/src/integration_test.cc

```diff
@@ -40,6 +41,17 @@
 #define FIREBASE_CONFIG_STRING ""
 #endif  // FIREBASE_CONFIG
 
+// Allow integration tests to enable retrying of file-not-found errors.
+#if FIREBASE_PLATFORM_DESKTOP
+namespace firebase {
+namespace storage {
+namespace internal {
+extern bool g_retry_file_not_found__for_testing;
```

Comment:
nit: there's an extra underscore between "found" and "for"

Reply:
fixed